tests/resample/test_resample.py

```diff
@@ -153,7 +148,7 @@ def test_resample_compute_error_mode(compute_err, weight_type):
     resample = Resample(
         n_input_models=nmodels,
         output_wcs=output_model,
-        weight_type=weight_type,
+        weight_type="ivm",
```

Comment:
Why is this testing dropping using "exptime"? That's listed as a supported weight time:
https://github.com/spacetelescope/stcal/blob/615aa636877f37bb1a61131539125024c068544c/src/stcal/resample/resample.py#L114

Reply:
This test's main focus is testing error modes, not weights. Weights are computed independently from error computation mode and are tested elsewhere, e.g., `test_resample_mostly_defaults`. It is simply an unnecessary test.